include/s3select.h

```diff
@@ -2130,6 +2121,8 @@ class csv_object : public base_s3object
       m_previous_line = false;
       m_skip_first_line = true;
 
+      csv_string = csv_string.substr(tmp_buff.length() + 1);
+
       run_s3select_on_object(result, merge_line.c_str(), merge_line.length(), false, false, false);
```

Comment:
this call runs the merged line (previously broken line, with a current broken line)
no need to assign csv_string